fizzbuzz2.rb

```diff
@@ -0,0 +1,13 @@
+range = 1..20
+range.each do |num|
```

Comment:
OKなんですが、今回のケースであればわざわざ変数に入れるまでもなさそうかなと思いました。

```suggestion
(1..20).each do |num|
```